packages/expo-dev-launcher/android/src/main/java/expo/modules/devlauncher/network/DevLauncherNetworkLogger.kt

```diff
@@ -46,9 +46,9 @@ class DevLauncherNetworkLogger private constructor() {
   /**
    * Emits CDP `Network.requestWillBeSent` event
    */
-  fun emitNetworkWillBeSent(request: Request, requestId: String) {
+  fun emitNetworkWillBeSent(request: Request, requestId: String, redirectResponse: Response?) {
     val now = BigDecimal(System.currentTimeMillis() / 1000.0).setScale(3, RoundingMode.CEILING)
-    val params = mapOf(
+    var params = mutableMapOf(
```

Comment:
Maybe it's not the best place to introduce the `buildMap` helper function so you can leave it as it's. However, in a case like this - when you're changing from `mapOf` to `mutableMapOf` and all operations are done in one scope, it's better to use the `buildMap` function instead. Here is the documentation: https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.collections/build-map.html.